layers/sessions.go

```diff
@@ -79,8 +82,9 @@ func (s *SessionsLayer) onResponseError(ctx *layers.Context) error {
 		close(channelUntyped.(chan<- string))
 	}
 
-	brokenSessionID := ctx.ResponseHeaders.GetLast("x-crawlera-session").Value()
-	mgr.getBrokenSessionChan() <- brokenSessionID
+	if brokenSessionID := ctx.ResponseHeaders.GetLast("x-crawlera-session").Value(); brokenSessionID != "" {
+		mgr.getBrokenSessionChan() <- brokenSessionID
```

Comment:
Do you really need `;` here?

Reply:
The same syntax is used in other lines with `if ` statements.
Also it is used in official specs - [link](https://go.dev/ref/spec#If_statements)